section_to_course/compat.py

```diff
@@ -76,7 +76,11 @@ def block_key_class():
     """
     Get the BlockKey class from upstream.
     """
-    from xmodule.modulestore.split_mongo import BlockKey
+    try:
+        # Quince and newer
```

Comment:
Are you sure? [The CI](https://github.com/open-craft/section-to-course/actions/runs/7953617157/job/21709847144?pr=53) was passing for Quince. Shouldn't it be "Releases newer than Quince"?